ironfish/src/metrics/metricsMonitor.ts

```diff
@@ -21,6 +22,8 @@ export class MetricsMonitor {
   readonly p2p_OutboundTraffic_WebRTC: Meter
   readonly p2p_PeersCount: Gauge
 
+  readonly p2p_OutboundTrafficByPeer: Map<Identity, Meter> = new Map()
```

Comment:
nit: This is messages, not bandwidth right? Can we name this something to indicate its not bandwidth like our other ones. Maybe `OutboundMessagesByPeer`

Reply:
Yep I can do that 👍🏻